record-and-playback/core/scripts/post_events/post_events_analytics.rb.example

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/ruby
```

Comment:
Do we have a standard way to run ruby scripts from this directory? Is there a systemd config file and a worker to run it?